code/game/turfs/simulated/walls.dm

```diff
@@ -111,6 +112,12 @@
 
 	return
 
+/turf/simulated/wall/proc/adjacent_fire_act(turf/simulated/wall, radiated_temperature)
+	if(radiated_temperature > max_temperature)
+		take_damage(rand(10, 20) * (radiated_temperature / max_temperature))
+
+	return ..()

```

Comment:
BYOND doesn't have an internal `adjacent_fire_act` proc, so this shouldn't be here.